src/test/java/gov/healthit/chpl/aqa/pageObjects/ChplResourcesDropdownPage.java

```diff
@@ -0,0 +1,30 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+/**
+ * Class ChplResourcesDropDown definition.
+ */
+public final class ChplResourcesDropdownPage {
```

Comment:
Is this the actual navigation element where it drops down the resources section? If so, I think this is one of the few things that should be in the BasePage Page object, as it's likely usable on 95%+ of the pages